### PR TITLE
 Replace Blacklist and Whitelist with Safelist and Blocklist

### LINKED ISSUE #549: Replace Blacklist and Whitelist with Safelist and Blocklist

### PR DESCRIPTION
**Pull request by akornich**
- chore: resolve #549

### CODE REVIEW

**Comment by akornich** [comment]
@mrunalk , @christopherseaman , @waltjones , can you, guys, review this PR, please?

File: Rollbar/IRollbarConfig.cs [hunk 1]
```diff
@@ -171,6 +181,7 @@ public interface IRollbarConfig
         /// The fields mentioned in this list are guaranteed to be excluded 
         /// from the ScrubFields list in cases when the lists overlap.
         /// </remarks>
+        [Obsolete("Use the ScrubSafelistFields property instead.")]
```
Comment (waltjones) [inline Rollbar/IRollbarConfig.cs:184]: :+1:

File: UnitTest.Rollbar/RollbarConfigFixture.cs [hunk 2]
```diff
@@ -76,7 +76,7 @@ public void TestGetSafeScrubFields()
                 {
                     Environment = RollbarUnitTestSettings.Environment,
                     ScrubFields = scrubFields,
-                    ScrubWhitelistFields = scrubWhitelistFields,
+                    ScrubSafelistFields = scrubWhitelistFields,
```
Comment (waltjones) [inline UnitTest.Rollbar/RollbarConfigFixture.cs:79]: Rename the local var too?

Comment (akornich) [reply UnitTest.Rollbar/RollbarConfigFixture.cs:79]: yes, absolutely! thank you!

Comment (akornich) [reply UnitTest.Rollbar/RollbarConfigFixture.cs]: fixed

**Pull request merged**